logging_topic: /realtime_loop/mechanism_controller/feedback
toggle_logging_service: /data_logger/toggle
log_directory: mechanism/tests
log_type: mechanism